public/app/features/explore/QueryRow.tsx

```diff
@@ -33,6 +33,20 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
     textEditModeEnabled: false,
   };
 
+  onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Hmm... this is actually an `Explore` component and I think Dashboard editors are not wrapped with it. Does Shift+Enter still work in dashboards?

Reply:
Hmm it seems that <kbd>Shift</kbd>+<kbd>Enter</kbd> doesn't work in dashboards. Did it work before?